packages/common/y-provider/src/lazy-provider.ts

```diff
@@ -154,7 +154,7 @@ export const createLazyProvider = (
       encodeStateAsUpdate(doc, remoteUpdate ? remoteUpdate.state : undefined)
     );
 
-    doc.emit('sync', []);
+    doc.emit('sync', [true, doc]);
```

Comment:
@EYHN `doc.emit('sync', [true, doc]);` not sure the first param should be `false` or `true`

Reply:
should be `true`